detox/triage/history.js

```
@@ -0,0 +1,149 @@
+
+            // A cluster where every identified member was already failing on the
+            // baseline branch is a main regression, and no rerun will change that.
+            // Requiring *every* member rather than any avoids calling a mixed
+            // cluster pre-existing on the strength of one stale test.
+            all_failing_on_baseline: entries.length > 0 && entries.every(
+                (e) => e.history && e.history.failing_since_commit,
+            ),
+            any_failing_elsewhere: entries.some(
+                (e) => e.failing_elsewhere && e.failing_elsewhere.distinct_prs > 0,
+            ),
+            // An unreachable amnesty endpoint counts as exhausted.
```

ESLint

[error] Expected line before comment.

(lines-around-comment)